src/main/scala/org/ergoplatform/modifiers/history/Extension.scala

```diff
@@ -38,7 +39,7 @@ case class Extension(headerId: ModifierId,
   override def serializer: Serializer[Extension] = ExtensionSerializer
 
   override def toString: String = {
-    s"Extension(${Algos.encode(headerId)}, " +
+    s"Extension($headerId, " +
```

Comment:
we use `Algos.encode` here to be able to easier change `ModifierId` type from `String` to some byte wrapper